visual/icons/DragHandle.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="20" height="16" xmlns="http://www.w3.org/2000/svg">
+	<g fill="#def">
+		<circle cx="3" cy="4" r="2" />
+		<circle cx="10" cy="4" r="2" />
+		<circle cx="17" cy="4" r="2" />
+		<circle cx="3" cy="12" r="2" />
+		<circle cx="10" cy="12" r="2" />
+		<circle cx="17" cy="12" r="2" />
+	</g>
+</svg>
```

Comment:
```suggestion
<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><g fill="#def"><circle cx="4" cy="4" r="2"/><circle cx="12" cy="4" r="2"/><circle cx="12" cy="20" r="2"/><circle cx="4" cy="12" r="2"/><circle cx="4" cy="20" r="2"/><circle cx="12" cy="12" r="2"/></g></svg>
```

I'd prefer that we use an SVG like this. I've explained in other places how I'd like for it to be used. Also, "handle" has other uses within the codebase, so name this file GripDots, GripPattern, GrabberDots, something like that.